src/run/run.c

```diff
@@ -174,7 +174,8 @@ static int help(void) {
                "  -P --pipe                       Pass STDIN/STDOUT/STDERR directly to service\n"
                "  -q --quiet                      Suppress information messages during runtime\n"
                "  -v --verbose                    Show unit logs while executing operation\n"
-               "     --json=pretty|short|off      Print unit name and invocation id as JSON\n"
+               "     --json=pretty|short|off      Print trigger unit name, unit name and\n"
+               "                                  invocation id as JSON, as applicable\n"
```

Comment:
Oxford comma, i.e. add a comma before "and". But maybe this level of detail is not necessary here. I think "Show information about created units as JSON" would be enough?